packages/runtime/src/core.ts

```diff
@@ -102,6 +104,16 @@ export class FederationHost {
       ],
       HTMLLinkElement | void
     >(),
+    loadEntry: new AsyncHook<
+      [
+        {
+          origin: FederationHost;
```

Comment:
it seems that `origin` only be used for `origin.loaderHook.lifecycle.createScript` and `origin.loaderHook.lifecycle.loadEntry` , so i prefer to just pass the hook , instead of originHost . 

WDYT? @zhoushaw

Reply:
Okay i have talked with him , and we think you can move `loadEntry` to `remoteHandler` hook